config/locales/cy.yml

```diff
@@ -1,6 +1,11 @@
 cy:
   language_name: Cymraeg
 
+  layouts:
+    base:
+      provided_by: Pension Wise is provided by
```

Comment:
@arwelowen Could you please translate this text: 'Pension Wise is provided by'? Thanks.